### PR TITLE
Log only diff Slurm jobs stdout/stderr

### LINKED ISSUE #397: Slurm job logs are displayed many time
# Bug Report

## Description
When Slurm job is monitored, the Slurm StdOut/SrdErr are logged entirely each time.

## Expected behavior
Only a diff is logged instead of the entire log

### PR DESCRIPTION
**Pull request by stebenoist**
# Pull Request description

### What I did

- Use updated action lastIndex to store the last read stdout or stderr log line
- Exec tail -n +x to read the new lines with x=lastIndex+1

### How to verify it

Run Tosca-samples Jobs

### Description for the changelog

* Slurm job logs are displayed many time ([GH-397](https://github.com/ystia/yorc/issues/397))

## Applicable Issues

Fixes #397

### CODE REVIEW

File: prov/slurm/monitoring_jobs.go [hunk 1]
```diff
@@ -189,4 +199,31 @@ func (o *actionOperator) logFile(ctx context.Context, deploymentID, filePath, fi
 		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelINFO, deploymentID).RegisterAsString(fmt.Sprintf("%s %s:", fileType, filePath))
 		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelINFO, deploymentID).RegisterAsString("\n" + output)
 	}
+
+	// Update the last index
+	cc, err := cfg.GetConsulClient()
+	if err != nil {
+		log.Debugf("fail to retrieve consul client due to error:%+v:", filePath, err)
```
Comment (laurentganne) [inline prov/slurm/monitoring_jobs.go]: unexpected filePath argument

Comment (stebenoist) [reply prov/slurm/monitoring_jobs.go]: oups

Comment (stebenoist) [reply prov/slurm/monitoring_jobs.go]: You're stronger than codeClimate !

Comment (laurentganne) [reply prov/slurm/monitoring_jobs.go]: :-)

**Comment by stebenoist** [comment]
So, you just need to test ? http://129.183.140.239:8088

**Comment by laurentganne** [comment]
> 
> 
> So, you just need to test ? http://129.183.140.239:8088

Works great!

**Pull request merged**